std/random.d

```diff
@@ -1782,7 +1782,7 @@ version (linux)
 
             `getrandom()` was added to the GNU C Library in v2.25.
          +/
-        pragma(msg, "`getrandom()` emulation for legacy Linux targets is enabled.");
+        //pragma(msg, "`getrandom()` emulation for legacy Linux targets is enabled.");
```

Comment:
Is there a reason for leaving this commented out instead of just removing it?

Reply:
Nothing too substantial.